app/src/main/java/org/simple/clinic/bp/BloodPressureModule.kt

```diff
@@ -50,7 +50,7 @@ open class BloodPressureModule {
   open fun provideBloodPressureEntryConfig(): BloodPressureConfig {
     return BloodPressureConfig(
         deleteBloodPressureFeatureEnabled = false,
-        dateEntryEnabled = false)
+        dateEntryEnabled = true)
```

Comment:
Can we leave this disabled for now until we make the changes to inject the system time zone?

Reply:
See my comment?

https://github.com/simpledotorg/simple-android/pull/460/commits/e9722f3bb9718e481c0b49a4f354f1b66dc46f17#r251286760

Reply:
> See my comment?
> 
> [e9722f3#r251286760](https://github.com/simpledotorg/simple-android/commit/e9722f3bb9718e481c0b49a4f354f1b66dc46f17#r251286760)

This doesn't link to any comment.

Reply:
Do you still want me to do this? Considering using the user's timezone in `UserInputDateValidator` can go in a separate PR?

Reply:
Are we going to make that change immediately? If we are, then we can skip this for now.